{"name": "test_men_age_gender_recognition[CPU]", "status": "failed", "statusDetails": {"message": "AssertionError: \nExpected: a value less than <0.5>\n     but: was <0.67640907>", "trace": "device_name = 'CPU'\n\n    @pytest.mark.smoke\n    @pytest.mark.parametrize(\"device_name\", [\"CPU\"])\n    def test_men_age_gender_recognition(device_name):\n        age_gender_model_run = AgeGenderRecognitionRetailModelRun(\"age-gender-recognition-retail-0013\", device_name)\n        net_PVB, exec_net_PVB = age_gender_model_run.model_init()\n        model_result = age_gender_model_run.get_model_run_output(cv.imread(os.path.dirname(os.path.realpath(__file__))\n                                                                           + '/women_face.jpeg'), net_PVB, exec_net_PVB)\n>       assert_that(model_result['prob'][0, 0, 0, 0], less_than(0.5))\nE       AssertionError: \nE       Expected: a value less than <0.5>\nE            but: was <0.67640907>\n\ntests/test_age_gender_recognition_model/test_age_gender_recognition_model.py:29: AssertionError"}, "parameters": [{"name": "device_name", "value": "'CPU'"}], "start": 1656931721653, "stop": 1656931721715, "uuid": "bc578029-0233-4017-bb44-09963de0baf9", "historyId": "e087eddb28e7d0c0c8d67f9412e53737", "testCaseId": "59faf519daea54807339b6704c5dde4c", "fullName": "tests.test_age_gender_recognition_model.test_age_gender_recognition_model#test_men_age_gender_recognition", "labels": [{"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "tests.test_age_gender_recognition_model"}, {"name": "suite", "value": "test_age_gender_recognition_model"}, {"name": "host", "value": "KVL2106006"}, {"name": "thread", "value": "31052-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_age_gender_recognition_model.test_age_gender_recognition_model"}]}